.github/workflows/coverage.yml

```diff
@@ -26,10 +26,7 @@ jobs:
           toolchain: nightly-2022-08-11
```

Comment:
And for the coverage CI, probably need to upgrade the nightly version to a more recent version such as 2023-03-09